electron/games.ts

```diff
@@ -264,6 +267,17 @@ class LegendaryGame implements Game {
     return await execAsync(command)
   }
 
+  public async launchWin(){
+    const {
+      launcherArgs = ''
+
+    } = await this.getSettings()
+    const command = `${legendaryBin} launch ${this.appName} ${launcherArgs}`
+    console.log('\n Launch Command:', command)
+
+    return await execAsync(command)
+  }
+
   public async launch() {
```

Comment:
I would suggest checking `isWindows` here instead of the caller side.

Reply:
Ok